pages/linux/pvdisplay.md

```diff
@@ -0,0 +1,13 @@
+# pvdisplay
+
+> Display information about Logical Volume Manager (LVM) physical volumes.
+> See also: `lvm`.
+> More information: <https://man7.org/linux/man-pages/man8/pvdisplay.8.html>.
+
+- Display information about all physical volumes:
+
+`sudo pvdisplay`
+
+- Display information about the physical volume on `/dev/sda1`:
+
+`sudo pvdisplay {{/dev/sda1}}`
```

Comment:
@bl-ue Should we cange the device names here as proposed in #5036?

Reply:
I think probably so, but undecided. https://github.com/tldr-pages/tldr/pull/5328#discussion_r584900364

We should push that PR forward.